deployment/README.md

```diff
@@ -1 +1,47 @@
 # Deployment
+
+## Requirements
+
+* Helm
+* [helm-diff plugin](https://github.com/databus23/helm-diff)
+* helmfile
+
+## Helm
+
+Move on to your helmfile environment, for example local and then apply your changes:
+> cd /helmfile/local
+>
+> helmfile -f helmfile.yaml apply
```

Comment:
I don't understand this step. I don't have any helmfile environment :/
Perhaps, this step meant to configure helm locally, if so, perhaps it's easier to add a link to this blog https://www.openshift.com/blog/getting-started-helm-openshift which explains how to fully configure helm to be used with OpenShift.

Reply:
Did you make any changes related to this comment?